packages/stratimikos/cmake/Dependencies.cmake

```diff
@@ -2,9 +2,15 @@
 # adapters will get enabled by default.  Note that Belos and ML do *not* have
 # a required dependence on Epetra but the Stratimikos Belos and ML adapters
 # need the Thyra/Epetra adapters.
-SET(LIB_REQUIRED_DEP_PACKAGES ThyraEpetraAdapters)
-SET(LIB_OPTIONAL_DEP_PACKAGES Amesos AztecOO Belos Ifpack ML EpetraExt)
-SET(TEST_REQUIRED_DEP_PACKAGES)
+IF (Stratimikos_MAKE_EPETRA_OPTIONAL)
```

Comment:
okay great ! I thought you asked for this flag in #314, but I'll gladly remove it. (done in 293fa20)